stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ChatHelper.kt

```diff
@@ -46,7 +45,6 @@ object ChatHelper {
         ChatClient.Builder(apiKey, context)
             .notifications(notificationConfig, notificationHandler)
             .withPlugin(offlinePlugin)
-            .withErrorHandlers(OfflineErrorHandlerFactoriesProvider.createErrorHandlerFactories())
```

Comment:
Should we remove the error handlers here?

Reply:
I think for now we can setup them up in `OfflinePluginFactory` for simplification and then decide what to do next